gmsh_utils/gmsh_IO.py

```diff
@@ -253,12 +259,13 @@ def make_surface(self, line_list: Union[List[int], npt.NDArray[np.int_]], name_l
         Returns:
             int: The surface tag.
         """
-
-        surface = self.create_surface(line_list, name_label)
-
+        if name_label != "":
```

Comment:
this if statement is unnecessary. Without the statement, exactly the same happens